### ISSUE TITLE
FontForge stopped suddenly

### ISSUE THREAD

**Issue by suman51284** [fontforge/fontforge#2412]
I cant open the software anymore on my laptop. tried reinstalling it several times since yesterday but in vain. Attached is the notification that pops up every time I try to install the program.
![untitled](https://cloud.githubusercontent.com/assets/11287162/8569862/60bb8f0e-2599-11e5-8915-a33a0d6c6e7d.jpg)

**Comment by adrientetar** [comment]
It seems there are fontforge processes running on your machine... try to kill them or reboot and try again.

**Comment by suman51284** [comment]
There are no FontForge processes running. I checked it. on further installing the program it is now showing this notification.
![untitled](https://cloud.githubusercontent.com/assets/11287162/8593844/ec037eae-2658-11e5-99c2-d4b09a76196a.jpg)

**Comment by adrientetar** [comment]
Have you tried to run as administrator?

**Comment by suman51284** [comment]
Yes, I have tried as well. I think I need to reboot the laptop. Will let you know by tomorrow morning if it doesn't run even then.
Thanks :)

**Comment by pathumego** [comment]
@suman51284 Are you using the latest release? if not what release are you using and on what Win system?

**Comment by suman51284** [comment]
Hi Pathum,
 I am using FontForge-2015-06-12-Windows, on the Windows 7, Ultimate OS.

**Comment by jtanx** [comment]
@suman51284, what antivirus software are you running?

**Comment by suman51284** [comment]
I am using Quick Heal antivirus.

**Comment by jtanx** [comment]
Are you sure your antivirus software is not blocking you from running FontForge? You shouldn't need administrator privileges to run FontForge.

**Comment by jtanx** [comment]
Oh yeah, just double check that the following processes are not running in Task Manager: `fontforge.exe`, `vcxsrv.exe` and `run_fontforge.exe`

**Comment by suman51284** [comment]
I tried turning the antivirus off while installing as well. It shows this notification before I start running it
![untitled](https://cloud.githubusercontent.com/assets/11287162/8595116/1e871c62-2665-11e5-8c5e-e75f26f0461b.jpg)

**Comment by jtanx** [comment]
That's to be expected, because I/we don't sign the binaries.

**Comment by suman51284** [comment]
I had cleared the task manager right after uninstalling the software.

**Comment by jtanx** [comment]
What warning/error message appears when you try to run `C:\Program Files (x86)\FontForgeBuilds\run_fontforge.exe` directly?

**Comment by suman51284** [comment]
This is the notification that pops up after running the program.
![ec037eae-2658-11e5-99c2-d4b09a76196a](https://cloud.githubusercontent.com/assets/11287162/8595650/872ca67c-2667-11e5-8a8f-7fd35725480a.jpg)

**Comment by jtanx** [comment]
That looks like the error message you get when you try to launch FontForge from the installer - have you tried launching FontForge from the Start Menu, or just going to `C:\Program Files (x86)\FontForgeBuilds` and double-clicking on `run_fontforge.exe`?

**Comment by suman51284** [comment]
It's finally running. Thank you all..:)

**Comment by jtanx** [comment]
That's great - can you explain how you got it to work for the benefit of anyone else who has this problem?

**Comment by suman51284** [comment]
The last time I was working on it, I was trying to interpolate a font. After I had created a bold version of the font, I tried to open it and the program won't run any more.
 Tried to reinstall the program and tried to run it as an administrator file as well. It wouldn't launch. I deleted the old FontForgeBuilds file completely before running the .exe file again and launching the program. It started working.
